eng/Analyzers_NonShippingRules.ruleset

```diff
@@ -1,40 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="Common diagnostic rules for all non-shipping projects" Description="Enables/disable rules specific to all non-shipping projects." ToolsVersion="14.0">
-  <!-- Include all shipping Roslyn-Analyzers project build rules -->
-  <Include Path=".\Analyzers_ShippingRules.ruleset" Action="Default" />
-
-  <Rules AnalyzerId="Microsoft.ApiDesignGuidelines.Analyzers" RuleNamespace="Microsoft.ApiDesignGuidelines.Analyzers">
-    <!-- Do not declare static members on generic types (we use generic static classes in test helpers) -->
-    <Rule Id="CA1000" Action="None" />
-    <!-- Nested types should not be visible (we use nested types as test helpers) -->
-    <Rule Id="CA1034" Action="None" />
-    <!-- IdentifiersShouldNotContainUnderscores - test names contain underscores and this is fine -->
-    <Rule Id="CA1707" Action="None" />
-    <!-- Type names should not match namespaces (not needed for test code) -->
-    <Rule Id="CA1724" Action="None" />
-    <!-- Do not directly await a Task (makes tests unnecessarily noisy) -->
-    <Rule Id="CA2007" Action="None" />
-  </Rules>
-  
-  <Rules AnalyzerId="Microsoft.CodeQuality.Analyzers" RuleNamespace="Microsoft.CodeQuality.Analyzers.Maintainability">
-    <!-- Mark member as static rule has many violations in non-shipping projects -->
-    <Rule Id="CA1822" Action="None" />
-
-    <!-- Do not directly await a task - not useful for tests -->
-    <Rule Id="CA2007" Action="None" />
-  </Rules>
-
-  <Rules AnalyzerId="Microsoft.CodeAnalysis.Features" RuleNamespace="Microsoft.CodeAnalysis.Features">
-    <!-- We restore IDE0005 to hidden severity for non-shipping test code as we have lot of autogenerated test files with helpful usings to help someone get started whenever they start authoring tests -->
-    <Rule Id="IDE0005" Action="Hidden" /> <!-- Remove unused imports/usings -->
-  </Rules>
-
-  <Rules AnalyzerId="xunit.analyzers" RuleNamespace="Xunit.Analyzers">
-    <Rule Id="xUnit1004" Action="Hidden" /> <!-- allow skipped tests, but also enable the refactoring to enable them -->
```

Comment:
Default is `Info`:

https://github.com/xunit/xunit.analyzers/blob/3783365c65a8377c51c73fbb7381df09cd02372d/src/xunit.analyzers/Utilities/Descriptors.cs#L79

So I dropped this.